From 36a6b35b5eb2874a85b5bc1caf111213c0bee04b Mon Sep 17 00:00:00 2001
From: Carlos Rafael Giani <dv@pseudoterminal.org>
Date: Fri, 29 Apr 2016 08:52:12 +0200
Subject: [PATCH] Workaround for glib-related build error with ozone-wayland

This fixes the following compilation error:

../../third_party/libsecret/secret.h:18:18: fatal error: glib.h: No such file or directory

Signed-off-by: Carlos Rafael Giani <dv@pseudoterminal.org>
---
 chrome/chrome_browser.gypi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/chrome/chrome_browser.gypi b/chrome/chrome_browser.gypi
index d964fec..daccacb 100644
--- a/chrome/chrome_browser.gypi
+++ b/chrome/chrome_browser.gypi
@@ -3543,6 +3543,9 @@
           ],
         }],
         ['OS=="linux" and chromeos!=1', {
+          'dependencies': [
+            '../build/linux/system.gyp:glib',
+          ],
           'sources': [ '<@(chrome_browser_libsecret_sources)' ],
           'defines': [
             'USE_LIBSECRET',
-- 
2.5.0

